### DISCUSSION TITLE
New version?

### DISCUSSION THREAD

**Discussion by avee87** [cioraneanu/firefly-pico#130] [General]
There have been a bunch of fixes in dev since the last regular release. Would it make sense to ship them as 1.4.3 if there is no plan for a next "major" version yet?

**Comment by cioraneanu** [comment]
I don't understand the impatience. :) Those fixes are already available for everyone to use. Just rebranding them as a separate tag doesn't change much. 
I want releases to feel like a true step forward that's why i'm waiting for more to add up. 

This period is extremely full and i have 0 free time, but hopefully things will clear up soon. 😇

**Reply by avee87** [reply]
Not everyone wants to use a dev tag. There is nothing wrong with patch releases like 1.4.2

**Comment by cioraneanu** [comment]
Published 1.5.0
